src/vm/appdomain.cpp

```diff
@@ -6411,6 +6417,9 @@ HRESULT RuntimeInvokeHostAssemblyResolver(INT_PTR pManagedAssemblyLoadContextToB
             //
             // Attempt to resolve it using the ResolveSatelliteAssembly method.
             // Finally, setup arguments for invocation
+            tracer.Trace(BinderTracing::ResolutionAttemptedOperation::Stage::DefaultALCFallback,
```

Comment:
I think this would be a new stage of `SatelliteAssembly` (though I don't think we need to include tracing for this stage in this change)

Reply:
Sure, will do as a followup patch.